react-cors-spa/src/Products.js

```diff
@@ -1,8 +1,25 @@
 import axios from 'axios';
 import { useQuery } from '@tanstack/react-query';
 import { DataGrid } from '@mui/x-data-grid';
+import { createTheme, ThemeProvider, CssBaseline } from '@mui/material';
 import { useMemo } from "react";
 
+
+const theme = createTheme({
```

Comment:
This needs to be in it's own file and folder

Reply:
And can I get another photo?

Reply:
Yes definitely my bad! totally forgot to update this PR with the screenshot:

![image](https://github.com/user-attachments/assets/4d2faa2a-814c-4cd4-8ac2-9d06c5ab1b32)